src/apis/avm/api.ts

```diff
@@ -783,6 +783,23 @@ class AVMAPI extends JRPCAPI{
         });
     }
 
+    /**
+     * Given a JSON representation of this Virtual Machine’s genesis state, create the byte representation of that state.
+     * 
+     * @param genesisData The blockchain's genesis data object
+     * 
+     * @returns Promise of a string a bytes
+     */
+    buildGenesis = async (genesisData:object):Promise<object> => {
```

Comment:
this is returning an object, not a string of bytes. Check the test in the tests folder, when you write it, make sure it checks for these issues. \